### PR TITLE
Updated downloads.js

### PR DESCRIPTION
**Pull request by PasunuriSrinidhi**
This will be the more advanced version and I think that the code provided looks well-structured and readable.

### CODE REVIEW

**Comment by srowen** [comment]
What changed here, mostly adding const? does it do anything?
You need to run jekyll to update the copy in site/
Maybe, just don't know if this matters.

**Comment by PasunuriSrinidhi** [comment]
> What changed here, mostly adding const? does it do anything? You need to run jekyll to update the copy in site/ Maybe, just don't know if this matters.

I converted normal function to es6 (Arrow function)

File: js/downloads.js [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
+const hadoop3p3 = {pretty: "Pre-built for Apache Hadoop 3.2 and later", tag: "hadoop3.2"};
+const hadoop3p3scala213 = {pretty: "Pre-built for Apache Hadoop 3.2 and later (Scala 2.13)", tag: "hadoop3.2-scala2.13"};
+const hadoop2p = {pretty: "Pre-built for Apache Hadoop 2.7", tag: "hadoop2"};
+const hadoop3p = {pretty: "Pre-built for Apache Hadoop 3.3 and later", tag: "hadoop3"};
+const hadoop3pscala213 = {pretty: "Pre-built for Apache Hadoop 3.3 and later (Scala 2.13)", tag: "hadoop3-scala2.13"};
+
+const packagesV12 = [hadoop3p3, hadoop3p3scala213, hadoop2p7, hadoopFree, sources];
+const packagesV13 = [hadoop3p, hadoop3pscala213, hadoop2p, hadoopFree, sources];
+
+addRelease("3.3.3", new Date("04/14/2023"), packagesV13, true);
```
Comment (xinrong-meng) [inline js/downloads.js:23]: I am wondering why `3.3.3` is associated with `04/14/2023`. The date should be the release date.

Comment (dongjoon-hyun) [reply js/downloads.js:23]: Ya, Apache Spark community doesn't release it yet.

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
Thank you for making a PR, @PasunuriSrinidhi . However, this PR looks wrong to me too.

**Pull request closed without merging**